### PR TITLE
OU-199: Use korrel8r to load logs related to an alert

### PR DESCRIPTION
**Pull request by kyoto**
With this PR, we just use the first korrel8r goal that has some LogQL results. In other words, if the korrel8r API returns multiple LogQL queries that have results, we ignore all but the first one in the API response.

https://github.com/openshift/logging-view-plugin/assets/460802/30a61144-82bc-42b2-adfb-2608065b1e31

### CODE REVIEW

File: web/src/hooks/useLogActionsExtension.tsx [hunk 1]
```diff
@@ -11,21 +12,73 @@ const useLogActionsExtension: ExtensionHook<Array<Action>, LogActionsExtensionOp
...
+                if (goal.class === 'application.log') {
+                  tenant = 'application';
+                } else if (goal.class === 'audit.log') {
+                  tenant = 'audit';
+                } else if (goal.class === 'infrastructure.log') {
+                  tenant = 'infrastructure';
+                }
+              }
+
+              if (logQL && tenant) {
```
Comment (jgbernalp) [inline web/src/hooks/useLogActionsExtension.tsx:49]: This probably can be outside the `response.forEach`, I assume we are searching in the response for the first goal that matches logs.

Comment (kyoto) [reply web/src/hooks/useLogActionsExtension.tsx]: Thanks @jgbernalp. I tidied up the function and made it more clear that we're just using the first goal that matches logs with this PR.

**Pull request merged**